ml-proto/test/block.wast

```diff
@@ -46,43 +46,62 @@
     (f32.gt (block (call $nop) (f32.const 3)) (block (call $nop) (f32.const 3)))
   )
 
-  (func "br-bare" (result i32)
+  (func "break-bare" (result i32)
```

Comment:
This is to match the loop test cases, where the names distinguish between breaking and continuing (see below). So the naming does not refer to the operator, but to the control flow transfer happening.